sdk/nodejs/iam/managedPolicies.ts

```diff
@@ -16,268 +16,532 @@
 import {ARN} from "../arn";
 
 export module ManagedPolicies {
+    /**@deprecated Use ManagedPolicy.AWSAccountActivityAccess instead. */
```

Comment:
Nit: add a space for these

```suggestion
    /** @deprecated Use ManagedPolicy.AWSAccountActivityAccess instead. */
```